.github/workflows/build.yaml

```
@@ -14,7 +14,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v4
         with:
-          go-version: 1.19
+          go-version: 1.21
```

Given that the required version is `1.22` (as stated in the `go.mod`) we should use the same version in the CI.

```suggestion
          go-version: 1.22
```

---

sure will do this change